projects/accumulator/index.js

```diff
@@ -0,0 +1,39 @@
+const utils = require("../helper/utils");
+const { toUSDTBalances } = require("../helper/balances");
+let _response;
+
+function fetchChain(chainId) {
+  return async () => {
+    if (!_response)
+      _response = utils.fetchURL("https://newapi.potluckprotocol.com/tvl");
```

Comment:
we prefer computing tvl on chain, is there some api to get vault list?

Reply:
Yes, its beefy API fork, and here is the list of our shimmer pools.

https://raw.githubusercontent.com/PotluckProtocol/potluck-vaults/main/src/features/configure/vault/shimmer_pools.js